morpheus/controllers/rss_controller.py

```diff
@@ -182,17 +206,25 @@ def parse_feeds(self):
             The parsed feed content.
         """
         for url in self._feed_input:
+            feed_stats: FeedStats = self._feed_stats_dict[url]
+            current_time = time.time()
             try:
-                if (url in self._errored_feeds):
-                    continue
+                if ((current_time - feed_stats.last_failure) >= self._cooldown_interval):
+                    feed = self._try_parse_feed(url)
 
-                feed = self._try_parse_feed(url)
+                    feed_stats.last_success = current_time
+                    feed_stats.success_count += 1
+                    feed_stats.last_try_result = "Success"
 
-                yield feed
+                    yield feed
 
             except Exception as ex:
-                logger.warning("Failed to parse feed: %s: %s. The feed will be not be retried.", url, ex)
-                self._errored_feeds.append(url)
+                logger.info("Failed to parse feed: %s: %s.", url, ex)
```

Comment:
I think leaving this as a warning is probably the right answer vs info, anyone monitoring the pipeline's output will probably want to see this.

Could you also update the string to include the number of failures and total attempts.